grandcypher/__init__.py

```diff
@@ -376,7 +376,7 @@ def _data_path_to_entity_name_attribute(data_path):
 
 class _GrandCypherTransformer(Transformer):
     def __init__(self, target_graph: nx.Graph, limit=None):
-        self._target_graph = target_graph
+        self._target_graph = nx.MultiDiGraph(target_graph)
```

Comment:
This is a super smart change and simplifies a ton — good thinking! there's probably a ton of business logic we can strip out as a result... thinking out loud, maybe makes sense to put in a test coverage library to auto-detect those chunks...

any performance hits you can think of as a result of doing this?

Reply:
> a test coverage library to auto-detect those chunks

that sounds like a great idea! Haven't used many test coverage libraries myself so open to suggestions :)

Also, w.r.t to performance hit I'm unsure about the impact of changing to `MultiDiGraph` -- at least in practice it appears to be similar. Probably a good idea to benchmark future versions

Reply:
I've been liking codspeed (e.g., https://github.com/aplbrain/grand/pull/48) — maybe a cool thing to extend to this repo someday!